tensorflow/lite/kernels/activations.cc

```diff
@@ -764,9 +764,10 @@ TfLiteStatus SoftmaxEval(TfLiteContext* context, TfLiteNode* node) {
         Softmax4DFloat(input, output, params);
         return kTfLiteOk;
       }
-      context->ReportError(
-          context, "Only 1D, 2D and 4D tensors supported currently, got %dD.",
-          NumDimensions(input));
+      context->ReportError(context,
+                           "Only 1D, 2D, 3D and 4D tensors supported currently "
+                           "Float32 kernels, got %dD.",
+                           NumDimensions(input));
```

Comment:
It will be more clear to change the wording to:
"The input tensor is 2D. Only 1D, 2D, 3D and 4D tensors are supported currently in Float32 kernel."

Same for the other logs.

Reply:
@lu-wang-g Input tensor can be any of dimension 1D, 2D, 3D and 4D right?
Should I add only "Only 1D, 2D, 3D and 4D tensors are supported currently in Float32 kernel, got %dD."